python/ql/src/semmle/python/frameworks/Stdlib.qll

```diff
@@ -1542,6 +1542,45 @@ private module Stdlib {
   class Sqlite3 extends PEP249Module {
     Sqlite3() { this = sqlite3() }
   }
+
+  // ---------------------------------------------------------------------------
+  // re
+  // ---------------------------------------------------------------------------
+  private module Re {
+    /** List of re methods. */
+    private class ReMethods extends string {
+      ReMethods() { this in ["match", "fullmatch", "search", "split", "findall", "finditer"] }
+    }
+
+    /** re.ReMethod(pattern, string) */
+    private class DirectRegex extends RegexExecution::Range {
+      DirectRegex() {
+        exists(ReMethods reMethod, DataFlow::CallCfgNode reCall |
+          reCall = API::moduleImport("re").getMember(reMethod).getACall() and
+          this = reCall.getArg(0)
+        )
+      }
+    }
+
+    /** re.compile(pattern).ReMethod */
+    private class CompiledRegex extends RegexExecution::Range {
+      CompiledRegex() {
+        exists(DataFlow::CallCfgNode patternCall, DataFlow::AttrRead reMethod |
+          patternCall = API::moduleImport("re").getMember("compile").getACall() and
+          patternCall = reMethod.getObject().getALocalSource() and
+          reMethod.getAttributeName() instanceof ReMethods and
+          this = patternCall.getArg(0)
+        )
+      }
+    }
+
+    private class RegexEscape extends DataFlow::Node {
+      RegexEscape() {
+        this =
+          API::moduleImport("re").getMember("escape").getACall().(DataFlow::CallCfgNode).getArg(0)
+      }
+    }
```

Comment:
Since we are not filling out a concept here, this just gets inlined into the configuration (so these lines get deleted).